src/fdcache.cpp

```diff
@@ -1999,7 +2003,7 @@ int FdEntity::RowFlush(const char* tpath, bool force_sync)
           return -1;
         }
         for(fdpage_list_t::const_iterator iter = dlpages.begin(); iter != dlpages.end(); ++iter){
-          if(0 != (result = Load(iter->offset, iter->bytes, true))){
+          if(0 != (result = Load(iter->offset, iter->bytes, true, true))){  // set loaded and modified flag
```

Comment:
Maybe use a pseudo-keyword argument, e.g., `/*is_modified_flag=*/ true`?  clang-tidy checks these: https://clang.llvm.org/extra/clang-tidy/checks/bugprone-argument-comment.html

Reply:
Thanks! I will add it and update soon.